content/en/agent/kubernetes/distributions.md

```diff
@@ -0,0 +1,303 @@
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  credentials:
+    apiKey: <DATADOG_API_KEY>
+    appKey: <DATADOG_APP_KEY>
+  agent:
+    config:
+      criSocket:
+        criSocketPath: /run/dockershim.sock
+  agent:
+    image:
+      name: "gcr.io/datadoghq/agent:latest"
+    apm:
+      enabled: false
+    process:
+      enabled: true
+      processCollection: false
+    log:
+      enabled: false
+    systemProbe:
+      enabled: false
+    security:
+      compliance:
+        enabled: false
+      runtime:
+        enabled: false
+  clusterAgent:
+    image:
+      name: "gcr.io/datadoghq/cluster-agent:latest"
+    config:
+      externalMetrics:
+        enabled: false
+      admissionController:
+        enabled: false
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Azure Kubernetes Service (AKS)
+
+AKS requires specific configuration for the `Kubelet` integration due to AKS certificates setup.
+
+**Note**: In some setups, DNS resolution for `spec.nodeName` inside Pods may not work in AKS. In this case, using `tlsVerify: false` is required.
+This has been reported on all AKS Windows nodes and occasionally on some Linux setups (mainly on Kubernetes < 1.18)
+
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+Custom `values.yaml`:
+
+```
+datadog:
+  apiKey: <DATADOG_API_KEY>
+  appKey: <DATADOG_APP_KEY>
+  kubelet:
+    host:
+      valueFrom:
+        fieldRef:
+          fieldPath: spec.nodeName
+    hostCAPath: /etc/kubernetes/certs/kubeletserver.crt
+    # tlsVerify: false # If Kubelet integration fails with provided configuration
+```
+
+{{% /tab %}}
+{{% tab "Operator" %}}
+
+Datadog Agent CR:
+
+```
+apiVersion: datadoghq.com/v1alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  credentials:
+    apiKey: <DATADOG_API_KEY>
+    appKey: <DATADOG_APP_KEY>
+  agent:
+    config:
+      kubelet:
+        host:
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        hostCAPath: /etc/kubernetes/certs/kubeletserver.crt
+        # tlsVerify: false # If Kubelet integration fails with provided configuration
+  agent:
+    image:
+      name: "gcr.io/datadoghq/agent:latest"
+    apm:
+      enabled: false
+    process:
+      enabled: true
+      processCollection: false
+    log:
+      enabled: false
+    systemProbe:
+      enabled: false
+    security:
+      compliance:
+        enabled: false
+      runtime:
+        enabled: false
+  clusterAgent:
+    image:
+      name: "gcr.io/datadoghq/cluster-agent:latest"
+    config:
+      externalMetrics:
+        enabled: false
+      admissionController:
+        enabled: false
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Google Kubernetes Engine (GKE)
+
+Since Agent 7.26, no specific configuration is required for GKE anymore (whether you run `Docker` and/or `containerd`).
+
+**Note**: When using COS (Container Optimized OS), the eBPF-based `OOM Kill` and `TCP Queue Length` checks are not supported due to missing Kernel headers.
```

Comment:
This is a good note to move from here to:
https://github.com/DataDog/integrations-core/blob/master/gke/README.md